### DISCUSSION TITLE
Add support for Darwinia on Openzeppelin Defender

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#85] [Integration]
https://docs.openzeppelin.com/defender/v2/

**Comment by hackfisher** [comment]
And better Crab chain and the testnet

**Comment by nblogist** [comment]
Looks like will have to message them using a feedback form, drafted the following, pls review @hackfisher and I'll send it!

------------------------------------------------------------------------------------------------------------------------------------------------------------------
Hello OpenZeppelin Defender Team, I am writing to request the addition of two networks, [Darwinia](https://darwinia.network/) and Crab Network, to the OpenZeppelin Defender platform. Our user base for these networks is growing, and we believe integrating them with Defender will not only significantly aid our developers but also potentially increase the usage of Defender across a broader range of networks.

Here are some specifics about each network:

Darwinia Network

Purpose and Use Case: #92 
Chain ID: **46**
RPC URL: **https://rpc.darwinia.network/**
Symbol: **RING**


Crab Network

Purpose and Use Case: #92 
Chain ID: **44**
RPC URL: **https://crab-rpc.darwinia.network/**
Symbol: **CRAB**

We are confident that adding these networks to Defender will enhance our development processes and contribute to the broader ecosystem. We look forward to potentially collaborating and expanding the capabilities of OpenZeppelin Defender.

Thank you for considering our request. We are eager to hear back from you regarding the next steps.

Best regards,
Furqan A.
Integrations Manager,
Darwinia Network.

**Reply by hackfisher** [reply]
Looks good to me. Would you please add my telegram, I can add you to a collaboration group with them, if you need reference etc.
